### PR TITLE
updated gitignore file to exclude sqlconfigs

### PR DESCRIPTION
**Pull request by muarachmann**
This should fix the defaults sqlconfig file from being tracked by git

### CODE REVIEW

**Comment by muarachmann** [comment]
@Ngai-E have a look when you can

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,6 @@
 .idea/
 sites/**
 !sites/default
-sites/default/sqlconfig
+sites/default/sqlconfig.php
```
Comment (Ngai-E) [inline .gitignore]: @muarachmann  the file `sqlconfig.php`  does not exist, I think you meant `sqlconf.php` as shown in '1' below
image

- for development purposes, users who had already installed this system will need to run  `git rm --cached .\sites\[path_to_active_site_name]\sqlconf.php`,  for example,  mine is (git rm --cached .\sites\default\sqlconf.php) before git can start ignoring the file.

**Comment by KoniKodes** [comment]
What is the status?

**Comment by mhn94** [comment]
This is one of the issues I'm assigned to work on. What's the status?

**Comment by Ngai-E** [comment]
> This is one of the issues I'm assigned to work on. What's the status?

Yes, go ahead. I do not see it in the codebase which means it is still pending.

**Comment by mhn94** [comment]
I've looked into this, and I don't think there's much for me to do. @muarachmann already added the correct file to .gitignore. Like @Ngai-E mentioned, people who already have the repo cloned before this change will have to run `git rm --cached ./sites/[path_to_active_site_name]/sqlconf.php` every time a new branch is created, unless they re-clone the repo.

**Comment by muarachmann** [comment]
Yes please work on another issue

**Comment by muarachmann** [comment]
Okay merging this troubler ...

**Pull request merged**